### DISCUSSION TITLE
Export to photoshop?

### DISCUSSION THREAD

**Discussion by DrkSlyr00** [dmMaze/BallonsTranslator#598] [Q&A]
Is there a way to export the translation project to Photoshop so that I have more freedom to edit, image layers, and text layers?

**Comment by bropines** [comment]
> Is there a way to export the translation project to Photoshop so that I have more freedom to edit, image layers, and text layers?

Yeah, you haven't read the readme at all. I'm looking into the matter. The problem is that photoshop itself doesn't have any debug modes, and so I can't finish the export to photoshop -_-

**Reply by DrkSlyr00** [reply]
Sorry (,,>﹏<,,), I already read it, It's a bit inconvenient but still your project is really great, thanks for your hard work... (づ ᴗ _ᴗ)づ♡

**Reply by bropines** [reply]
And in what form do you see the export to Photoshop? I had different ideas, but the problem is that PSD is a proprietary protocol, and so far we can only read it, but not write it ourselves in the form we want. I am still working on an extension that will allow me to work with projects from BT to PS in a normal way. And here's the question, whether to just do a json configuration exporter with subsequent overlay, or write an api service that will automatically pull up the project from BT.

**Reply by DrkSlyr00** [reply]
PSD
----Text layer
----Inpaint layer
----Image layer

Maybe you can use something of GIMP, is open source, i used it a little, it let you export to PSD, I don't remember if it let you export in that structure...

**Reply by bropines** [reply]
Well, if I understand how to write a plugin for gimp, then most likely I can do the export much easier, but you will have to have a gimp to create a PSD

**Comment by eadmaster** [comment]
what about using PDF?
It can be imported in Libreoffice and probably Photoshop too.

**Reply by bropines** [reply]
hmmmm... And as the idea is not bad, I'll see what I can do.
